examples/README.md

```diff
@@ -116,6 +116,7 @@ Example | File | Description
 `texture` | [`3d/texture.rs`](./3d/texture.rs) | Shows configuration of texture materials
 `update_gltf_scene` | [`3d/update_gltf_scene.rs`](./3d/update_gltf_scene.rs) | Update a scene from a gltf file, either by spawning the scene as a child of another entity, or by accessing the entities of the scene
 `wireframe` | [`3d/wireframe.rs`](./3d/wireframe.rs) | Showcases wireframe rendering
+`shapes` | [`3d/shapes.rs`](./3d/shapes.rs) | A scene showcasing more built-in shapes
```

Comment:
“more” compared to what?

Reply:
Oops, accidentally threw out Alice's suggestion there. That particular description made slightly more sense when it was listed directly below the 3d scene example.

Reply:
Yeah, in any case it's better to keep the descriptions devoid of their context, as it can change over time.